core/src/main/java/org/apache/accumulo/core/metadata/TabletFile.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.metadata;
+
+import org.apache.hadoop.fs.Path;
+
+public interface TabletFile {
+
+  /**
+   * @return The file name of the TabletFile
+   */
+  String getFileName();
+
+  /**
+   * @return The path of the TabletFile
+   */
+  Path getPath();
```

Comment:
Could you expand on these javadocs a bit to emphasize the difference between these two methods? It's not at all clear. Also, the `@return` is probably not really needed, if you just put a good main description.

Reply:
Created #3455 and assigned to you so it doesn't get lost. If you don't want to / can't do it, just unassign yourself, and somebody else can pick it up.